stdlib/SparseArrays/src/sparsevector.jl

```diff
@@ -1493,13 +1493,13 @@ function (*)(A::StridedMatrix{Ta}, x::AbstractSparseVector{Tx}) where {Ta,Tx}
     require_one_based_indexing(A, x)
     m, n = size(A)
     length(x) == n || throw(DimensionMismatch())
-    Ty = promote_op(matprod, Ta, Tx)
+    Ty = promote_op(matprod, eltype(A), eltype(x))
     y = Vector{Ty}(undef, m)
     mul!(y, A, x)
 end
 
 mul!(y::AbstractVector{Ty}, A::StridedMatrix, x::AbstractSparseVector{Tx}) where {Tx,Ty} =
-    mul!(y, A, x, one(Tx), zero(Ty))
+    mul!(y, A, x, 1, zero(Ty))
```

Comment:
Was this missed for the `true`/`false` transition?

Reply:
Yep!